assets/airports/terrain/kstl.geojson

```diff
@@ -0,0 +1,7 @@
+{
+"type": "FeatureCollection",
+"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:OGC:1.3:CRS84" } },
+"features": [
+
+]
```

Comment:
If I recall correctly, there was some terrain south-west of the field, just outside the airspace. But nothing above 1,000ft inside the TRACON.